packages/waku/src/lib/builder.ts

```diff
@@ -312,11 +312,14 @@ const emitHtmlFiles = async (
     config.rootDir,
     config.distDir,
     config.publicDir,
+    config.srcDir,
     config.indexHtml,
   );
-  const publicIndexHtml = fs.readFileSync(publicIndexHtmlFile, {
+  const publicIndexHtml = await fs.promises.readFile(publicIndexHtmlFile, {
     encoding: 'utf8',
   });
+
+  await fs.promises.unlink(publicIndexHtmlFile);
```

Comment:
Does this mean it creates two index html files?

Reply:
It creates one, gets it path, then emits it. That's how it was done before, but since it was the same path, it was getting overwritten as far as I know.